ui/src/components/core/button/button.utils.ts

```diff
@@ -0,0 +1,84 @@
+import { CSS } from '@stitches/react';
+
+export type ButtonColor = 'gray' | 'blue';
+export type ButtonVariant = 'solid' | 'outline' | 'ghost' | 'link';
+
+export type GetButtonCompoundVariantOptions = {
+  color?: ButtonColor;
+  variant?: ButtonVariant;
+};
+
+export const getButtonCompoundVariant = ({

```

Comment:
This seems to be style definition and will not be used in any other file than `button.styled.ts` so maybe it makes more sense to keep in the styling file.